ddtrace/contrib/cassandra/session.py

```diff
@@ -257,7 +259,7 @@ def _sanitize_query(span, query):
 # DEPRECATED
 #
 
-@deprecated(message='Use patching instead (see the docs).', version='0.6.0')
+@deprecated(message='Use patching instead (see the docs).', version='1.0.0')
```

Comment:
we can safely remove everything after going 1.0. No reason to remove these things now, even if it's reasonable safe.